vault/provider_test.go

```diff
@@ -118,23 +117,35 @@ func getTestAzureConf(t *testing.T) *azureTestConf {
 }
 
 func getTestGCPCreds(t *testing.T) (string, string) {
-	credentials := os.Getenv("GOOGLE_CREDENTIALS")
+	maybeCreds := os.Getenv("GOOGLE_CREDENTIALS")
 	project := os.Getenv("GOOGLE_PROJECT")
 
-	if credentials == "" {
+	if maybeCreds == "" {
 		t.Skip("GOOGLE_CREDENTIALS not set")
 	}
 
 	if project == "" {
 		t.Skip("GOOGLE_PROJECT not set")
 	}
 
-	contents, _, err := pathorcontents.Read(credentials)
-	if err != nil {
-		t.Fatal("Error reading GOOGLE_CREDENTIALS: " + err.Error())
+	maybeFilename := maybeCreds
```

Comment:
FWIW I would totally be ok with dropping support for this env var supporting _both_ contents and file path, and just picking one. But what you've got now is the better thing in the context of this PR 👍